dashboard/src/app/services/custom-interceptor.ts

```diff
@@ -9,9 +10,13 @@ export class CustomInterceptor implements HttpInterceptor {
   }
 
   intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    req = req.clone({
+    // console.log's for debugginggit add
+    console.log(req);
+    const apiReq = req.clone({
+      url: `https://${environment.apiUrl}${req.url}`,
       withCredentials: true
     });
-    return next.handle(req);
+    console.log(apiReq);
```

Comment:
We should remove the console logs when it comes to finalization

Reply:
sure